components/title.js

```diff
@@ -35,6 +37,7 @@ export default function Title(props) {
         }}
         className="align-middle inline-block"
       >
+        {/* Logo Section */}
         <div className="flex flex-col space-y-10">
           <div className="flex-grow mx-auto">
```

Comment:
merge `master` I had to remove `mx-auto` to get the title logo to display again.
Your current UI elements go off the page when the title logo is visible again.
![image](https://github.com/user-attachments/assets/1fd62022-2dab-437e-8e76-ebff4340ba07)

Reply:
Including the roomcode, the join url, title logo and the team names we need to fit everything on the page.

Probably need to join together elements or put elements togethor horizontially to fit everything.

Reply:
How do you like for the ui of the room code to look like this instead?
![image](https://github.com/user-attachments/assets/ff2708a1-f56e-4633-8a05-d5fc7bd8889c)
```jsx
          {/* Room Code Section */}
          <div className="flex flex-row justify-center text-center">
            <div className="flex flex-col items-center rounded-xl bg-secondary-500 shadow-lg p-6">
              <p className="text-6xl font-bold  text-foreground ">
                {props.game.room}
              </p>
              <p className="text-xl font-semibold text-foreground">
                {t("Room Code")}
              </p>
            </div>
          </div>
```

Reply:
You could join all the elements doing something like this
```jsx
          {/* Room Code Section */}
          <div className="flex flex-row justify-center text-center">
            <div className="flex flex-col items-center rounded-xl bg-secondary-500 shadow-lg p-6">
              <p className="text-6xl font-bold  text-foreground ">{props.game.room}</p>
              <p className="text-xl font-semibold text-foreground">{t('Room Code')}</p>
              {process.env.NEXT_PUBLIC_TITLE_URL && (
                <span className="text-4xl p-6 text-foreground rounded-xl bg-secondary-500 transition-colors duration-200">
                  {t('Join at')} <span className="font-bold">{process.env.NEXT_PUBLIC_TITLE_URL}</span>
                </span>
              )}
            </div>
          </div>
```
![image](https://github.com/user-attachments/assets/bcbd46ff-1d53-43cf-8298-3177e34ffceb)

Reply:
Thoughts about this?
![image](https://github.com/user-attachments/assets/8d493533-6666-4836-a8cf-42125c8f3df6)

And would you rather prefer the center blocks URL and code to be separated or be in one distinct div like in the picture you provided?

Reply:
This is pretty good. I personally like the 1 div approach. 

I think what I don't like is the "Room Code" description protruding on top of the room code. I would like if it was joined into 1 block.

Since you changed what the team list looks like. What does it look like when you have lots of team members?
You can go into the browser console and duplicate the list entry to display it.

Reply:
![image](https://github.com/user-attachments/assets/b8b1e24f-0b30-4233-b297-f7d54d6c5019)
Gave it a try. 
We could give it a minimum height so that it doesn't stretch the page down

```jsx
const Team = ({team, players}) => {
  return (
    <>
      <div className="flex flex-row text-center bg-secondary-500 rounded-xl shadow-lg overflow-hidden h-[200px]">
        <div className="flex-grow">
          <p className="text-4xl flex-grow text-foreground font-bold bg-secondary-700 rounded-t-xl py-3 shadow-sm">
            {team}
          </p>
          <div className="flex flex-wrap flex-row justify-center">
            {players.map((m) => (
              <div className="bg-primary-200 m-2 rounded-lg w-32 p-2">
                <p className="font-bold text-foreground overflow-hidden text-ellipsis whitespace-nowrap">{m}</p>
              </div>
            ))}
          </div>
        </div>
      </div>
    </>
  );
};

export default Team;
```

Reply:
![image](https://github.com/user-attachments/assets/7c7dcb4c-1098-4188-a8c4-7cc4e33c91ff)